### DISCUSSION TITLE
Python logging handler for Discord

### DISCUSSION THREAD

**Discussion by miohtama** [Rapptz/discord.py#7837] [General]
Hi all,

I have a written [new Python package that provides a Python logging handler for Discord](https://github.com/tradingstrategy-ai/python-logging-discord-handler).

It is good for

- Get notified on server-side errors
- Follow your batch job processes easily
- Good for businesses and communities that have their messaging set up in Discord
- Share logs with non-technical colleagues

Some features

- Minimum or no changes to a Python application needed 
- Optional colour coding of messages using [Discord embeds](https://discordjs.guide/popular-topics/embeds.html#embed-preview)
- Optional emoticons on messages using Unicode
- Discord rate-limiting friendly for a burst of logs
- Documentation
- Special handling for long log messages like tracebacks to deal with Discord's 2000 character max message length

The package is mean and lean. Unlike discord.py, the used API subset is much smaller - only webhook. 

Would there be any chance to include a link to `python-logging-discord-handler` in Discord.py documentation? This would improve the discoverability of the logging handler for those who need only a simple notification system or integrate Discord logging to their application.

**Comment by Rapptz** [comment]
Please don't advertise your library here -- likewise, my library's documentation does not advertise packages unless they're a strong recommendation from me or other contributors.
